src/acme/ui/AcmeUI.java

```diff
@@ -0,0 +1,151 @@
+package acme.ui;
+
+import acme.pd.*;
+
+import javax.swing.*;
+import java.awt.event.WindowEvent;
+import java.awt.event.WindowListener;
+
+public class AcmeUI extends JFrame {
```

Comment:
I was thinking on this. To make it easier to understand which UI classes are responsible for what, could we follow this convention?
* `XXXJPanel`
* `XXXJFrame`

**Examples**

* `AcmeUI` => `AcmeJFrame`
* `AcmeJPanelBase` => `AcmeJPanel`
* `ExampleJPanel`
* `UserUI` => `LoginJPanel`

Reply:
UserUI will be a Paresa change

Reply:
@pnoorossana